src/gui/statusbar.cpp

```diff
@@ -212,6 +216,12 @@ void StatusBar::updateDHTNodesNumber()
     }
 }
 
+void StatusBar::updateExternalAddressLabel()
+{
+    m_externalAddressLbl->setText(tr("External Address: %1").
+    arg(BitTorrent::Session::instance()->getExternalAddress()));
```

Comment:
```suggestion
    m_externalAddressLbl->setText(tr("External Address: %1")
            .arg(BitTorrent::Session::instance()->getExternalAddress()));
```

Reply:
Same as above: 'IP' instead of 'Address'

Reply:
@OdinVex, in the log when an external IP address is detected we have the string `Detected external IP`, I don't think we should move away from this.